.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
     fast_finish: true
     include:
         - php: 7.0
-          env: SYMFONY_VERSION=2.7.*
+          env: SYMFONY_VERSION=~2.7.2 # Workaround for a Composer bug
```

Comment:
I would rather update the `composer.json` instead: it's not that we prefer Sf 2.7.2, it's that it's not working with Sf 2.7.1 or 0 right?